tests/python_functional/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -46,8 +47,14 @@ def __init__(self, logger_factory, working_dir, version):
     def set_version(self, version):
         self.__syslog_ng_config["version"]=version
 
+    def set_raw_config(self, raw_config):
+        self.__raw_config = raw_config
+
     def write_content(self, config_path):
-        rendered_config = ConfigRenderer(self.__syslog_ng_config, self.__working_dir).get_rendered_config()
+        if self.__raw_config:
```

Comment:
I am agreed with you, it would be useful. But I am afraid this is not an easy change for now (to allow adding sub-parts of the config as a raw text). Later we can add this feature too.